### PR TITLE
EVG-16523 Use combobox for build variant selectors and task selectors

### PR DESCRIPTION
**Pull request by khelif96**
EVG-16523

### Description
Utilize the LG combobox component on the task and variant history pages
### Screenshots
image

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -4179,6 +4179,20 @@
     "@leafygreen-ui/typography" "^15.1.0"
     react-transition-group "^4.4.1"
 
+"@leafygreen-ui/checkbox@^12.0.4":
```
Comment (sophstad) [inline yarn.lock:4182]: This `yarn.lock` is out of date

File: src/pages/taskHistory/BuildVariantSelector.tsx [hunk 2]
```diff
@@ -48,35 +45,25 @@ const BuildVariantSelector: React.VFC<BuildVariantSelectorProps> = ({
...
-    );
-
   return (
     <Container>
-      <SearchableDropdown
+      <Combobox
         data-cy="build-variant-selector"
         label="Build Variant"
-        valuePlaceholder="Select build variant to view"
+        placeholder="Select build variant to view"
```
Comment (sophstad) [inline src/pages/taskHistory/BuildVariantSelector.tsx:53]: I know this was here before but "to view" feels redundant to me
```suggestion
        placeholder="Select build variants"
```

File: src/pages/taskHistory/BuildVariantSelector.tsx [hunk 3]
```diff
@@ -48,35 +45,25 @@ const BuildVariantSelector: React.VFC<BuildVariantSelectorProps> = ({
...
-    options.filter(
-      (option) =>
-        option.buildVariant.includes(match) ||
-        option.displayName.includes(match)
-    );
-
   return (
     <Container>
-      <SearchableDropdown
+      <Combobox
```
Comment (sophstad) [inline src/pages/taskHistory/BuildVariantSelector.tsx:50]: I assume this is a bug with the component but it's still annoying 😢 
https://user-images.githubusercontent.com/9298431/217654957-e368ed91-9acd-4a16-95f7-fd5a045ba6b5.mov

**Review by sophstad — APPROVED** [review]

File: src/pages/taskHistory/BuildVariantSelector.tsx (hunk 3 above)
Comment (khelif96) [reply src/pages/taskHistory/BuildVariantSelector.tsx:50]: I'll raise this with the LG folks

**Pull request merged**